opium_kernel/src/body.ml

```diff
@@ -40,16 +40,6 @@ let sexp_of_t { length; content } =
       ])
 ;;
 
-let string_of_t t =
```

Comment:
Deleted in favor of a pretty-printer, to have the same pair of functions `pp` and `pp_hum` as the other modules